### ISSUE TITLE
Is early stopping used for regularization?

### ISSUE THREAD

**Issue by ocissor** [yolish/multi-scene-pose-transformer#4]
Hi, I came across your paper and found it very interesting, while going through the code(main.py) I see that you have used weight decay but would like to know if you are saving the model w.r.t a validation loss or it is just based on the whether the epoch is divisible by a certain number. Thank you in advance.

**Comment by yolish** [comment]
Hi, thank you for your interest in our work and sorry for the delayed reply! Weight decay remains constant (based on the value set in the config file). The learning rate is decreased using a linear scheduler based on a predefined step size and gamma value (also defined in the config file). We have not explored other scheduling alternatives /performed thorough fine-tuning of the step size and gamma values/tried using a subset for validation loss as feedback. Regarding model saving -  the saving is done  just based on whether the epoch is divisible by a certain number. Hope this helps!
